libvirt.go

```diff
@@ -39,6 +39,21 @@ import (
 // are unsupported by either QEMU or libvirt.
 var ErrEventsNotSupported = errors.New("event monitor is not supported")
 
+// LibvirtURI defines a type for driver URIs for libvirt
+// the defined constants are *not* exhaustive as there are also options e.g. to connect remote via SSH
+type LibvirtURI string
```

Comment:
I'm not sure we gain much with an additional type here. Not a blocker for me, happy for others to weigh in.

Reply:
Thought it makes stuff easier for beginners to find the included constants with the IDE and type based predictions and because it's a string anyway it does not block anyone to provide custom URIs but I could of course change this?

Reply:
good enough for me :+1:

Reply:
I'm OK with using a type here, but the name stutters: it's `libvirt.LibvirtURI`. I suggest shortening the type name to just `URI`.